### PR TITLE
Fix stuff

### PR DESCRIPTION
**Pull request by skyem123**

### CODE REVIEW

**Comment by skyem123** [comment]
@NikkyAI could you go over the changes I've made to pack to ensure I'm using the updated voodoo correctly?

File: descentfrozenhell/config/railcraft/entities.cfg [hunk 1]
```diff
@@ -25,9 +25,9 @@ entities {
     B:cart_tnt_wood=true
     B:cart_trade_station=true
     B:cart_work=true
-    B:cart_worldspike_admin=false
-    B:cart_worldspike_personal=false
-    B:cart_worldspike_standard=false
+    B:cart_worldspike_admin=true
```
Comment (CitadelCore) [inline descentfrozenhell/config/railcraft/entities.cfg]: I don't see how this fixes the crash...

Comment (skyem123) [reply descentfrozenhell/config/railcraft/entities.cfg]: What is weird is that I got no crashes at all when I tested it.

Comment (skyem123) [reply descentfrozenhell/config/railcraft/entities.cfg]: Either it's fixed or its a bug that can only be found on the server... 
If it crashes I do have an idea on how to work around it, but we need to try this first.

Comment (ThePiGuy24) [reply descentfrozenhell/config/railcraft/entities.cfg]: I have attempted to test on the CTS but was unable to join due to "Fatally missing registry entries" as the config files kept getting overwritten locally.

Comment (skyem123) [reply descentfrozenhell/config/railcraft/entities.cfg]: Hm...

Comment (skyem123) [reply descentfrozenhell/config/railcraft/entities.cfg]: I tested it on a server and there were no crashes!

**Pull request merged**